students/11th/snagkuoh/js/main.js

```diff
@@ -19,39 +26,29 @@ replyArea.addEventListener("keydown", function(e) {
     }
 })
 
-// 버튼 활성화 //
+// 버튼 제어 //
 const btnOn = () => {
     if (replyArea.value) {
         replyBtn.disabled = false;
         replyBtn.classList ='reply-btn-active';
         replyBtn.classList.remove = 'reply-btn-off';
```

Comment:
```suggestion
        const { value } = replyArea
        replyBtn.disabled = !value;
        replyBtn.classList = value ? 'reply-btn-active' : 'reply-btn-off';
        replyBtn.classList.remove = value ? 'reply-btn-off' : 'reply-btn-active';
```
if 문 제거하고 이런 방식은 어떨까요?